x/marker/handler.go

```diff
@@ -73,8 +78,6 @@ func NewHandler(k keeper.Keeper) sdk.Handler {
 func NewProposalHandler(k keeper.Keeper) govtypesv1beta1.Handler {
 	return func(ctx sdk.Context, content govtypesv1beta1.Content) error {
 		switch c := content.(type) {
-		case *types.AddMarkerProposal:
-			return keeper.HandleAddMarkerProposal(ctx, k, c)
```

Comment:
This should probably be kept so that the old stuff continues to work. Then, in some later version, we can finally remove support for it.

Reply:
I initially agreed with your statement, but then I went back and looked at the old gov proposals on testnet and mainnet.  There hasn't been an AddMarkerProposal for a long time.  No one is actively using them.  So, I would say it is safe to remove.  When we start using it more there will just be the new way of doing the proposal.